res/deploy.tpl.php

```diff
@@ -34,6 +34,35 @@ function () {
 inventory( 'deploy.yml' );
 
 // Tasks.
+desc( 'Check WordPress is installed' );
+task(
+	'wp:maybe_installed',
```

Comment:
```suggestion
	'wp:install',
```

`maybe_installed` sounds like it only checks but does not install. Let's just name it `wp:install`?